library/envirophat/bmp280.py

```diff
@@ -153,15 +153,15 @@ def pressure(self):
         """Return the current air pressure."""
 
         self.update()
-        return self._pressure
+        return self._pressure / 10.0
```

Comment:
I believe that this should be `self._pressure / 100.0`, as hectopascal is 100 pascals.